README.md

```diff
@@ -97,6 +97,56 @@ $ make build
 $ make run-local
 ```
 
+## Local CLI installation
+
+You can install our CLI tool with the `pip`:
+
+```
+$ pip3 install --user git+https://github.com/packit-service/packit.git
+```
```

Comment:
```suggestion
```
Please bear in mind that packit works only with python 3.7 -- you're good if you are on Fedora 29 and later.
```

Reply:
Python 3.7 only? I think we can stay with 3.6 and resolve compatibility problems. Most of them will be fixed at #61 )